### DISCUSSION TITLE
Error 121 cannot logon with an Account

### DISCUSSION THREAD

**Discussion by manatwork81** [rustdesk/rustdesk-server-pro#359] [General]
Installed an new 24.04 Ubuntu with latest RD Server PRO.

All works but .... 

![rustdeskerror](https://github.com/user-attachments/assets/d62190e2-29df-4be1-9145-c1eb4accb702)

Any IDea?

THX

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#failed-to-log-in-with-custom-client-build-or-public-release-with-wrong-api-settings
